src/main/java/com/epam/reportportal/extension/azure/AzureExtension.java

```diff
@@ -184,4 +224,165 @@ private void removeListeners() {
 		pluginCommandMapping.put("testConnection", new TestConnectionCommand());
 		return pluginCommandMapping;
 	}
+
+	@Override
+    // Never called method. Connection is tested via the command.
+	public boolean testConnection(Integration system) {
+		return false;
+	}
+
+	@Override
+    // TODO: Implement in the future
+	public Optional<Ticket> getTicket(String id, Integration system) {
+		return Optional.empty();
+	}
+
+    @Override
+    // TODO: Implement in the future
+    public Ticket submitTicket(PostTicketRQ ticketRQ, Integration system) {
+        return null;
+    }
+
+	@Override
+	public List<PostFormField> getTicketFields(String issueType, Integration integration) {
+		Map<String, Object> params = integration.getParams().getParams();
+		String organizationUrl = params.get("url").toString();
+		String projectName = params.get("project").toString();
+		String personalAccessToken = params.get("oauthAccessKey").toString();
+
+		ApiClient defaultClient = getConfiguredApiClient(personalAccessToken);
+		String organizationName = organizationUrl.replace(defaultClient.getBasePath(), "");
+
+		ClassificationNodesApi nodesApi = new ClassificationNodesApi(defaultClient);
+		List<WorkItemClassificationNode> areaNodes = new ArrayList<>();
+		List<WorkItemClassificationNode> iterationNodes = new ArrayList<>();
+		try {
+            List<WorkItemClassificationNode> nodes = nodesApi
```

Comment:
Check you text editor settings and .editor config files. 
Indentation is broken here and in catch.
Program blocks are not clear.

Reply:
I think it is GitHub problem, i don't know why it shows code like this. The real code is well formatted.
![image](https://user-images.githubusercontent.com/68337846/129030835-5b7728a1-91d0-4538-b209-e8534041dcc6.png)

Reply:
Fixed